optimize_versioned_docs/version-3.9.0/self-managed/optimize-deployment/migration-update/3.9-preview-to-3.9.md

```diff
@@ -19,30 +19,33 @@ Here you will find information about:
 
 ## Known issues
 
-No known issues at the moment.
+If there are processes in Optimize that currently do not have a process owner assigned and a new process is deployed
+via Web Modeler, a new process owner may be assigned to one of the previous
+processes without an owner. This is not critical as this does not incur any changes in permissions, but is important to understand regarding who gets email notifications for processes. If an owner is set incorrectly, you can change it manually in the processes page.
+This issue is resolved with the 3.9.1 version.
 
 ## Changes in supported environments
 
-## Changes in translation files
+### Camunda Platform 7
 
-In case you manage your own translations into different languages, you can find a list below with all the changes that need to be translated for this release.
+Optimize now requires at least Camunda Platform 7 `7.16.0` and supports up to `7.18.0+`. Camunda Platform 7 `7.15.x` is not supported anymore.
+See the [supported environments]($docs$/reference/supported-environments/#camunda-platform-7--optimize-version-matrix) section for the full range of supported versions.
 
-### Localization file
+### Elasticsearch
 
-The following terms have been added/removed to/from the localization file (`en.json`) since the last release:
+Optimize now requires at least Elasticsearch `7.13.0`.
+See the [supported environments]($docs$/reference/supported-environments) section for the full range of supported versions.
 
-[en.json.diff](./translation-diffs/differences_localization_390_preview_1_390.diff)
+If you need to update your Elasticsearch cluster, refer to the general [Elasticsearch update guide](https://www.elastic.co/guide/en/elasticsearch/reference/current/setup-upgrade.html). Usually, the only thing you need to do is perform a [rolling update](https://www.elastic.co/guide/en/elasticsearch/reference/current/rolling-upgrades.html).
 
-- Lines with a `+` in the beginning mark the addition/update of a term; lines with a `-` mark the removal of a term.
+## Changes in translation files
 
-### Camunda Platform 7
+In case you manage your own translations into different languages, you can find a list below with all the changes that need to be translated for this release.
```

Comment:
"...you can find a list below with all the changes" -- Just need a little clarification here, what do you mean by this? Where is the list located or below what exactly?

Reply:
my suspicion is that this should be "you can find a diff below..." and this should be referring to the diff included in the Localisation file section. We probably only need one section here, but I will let Andrea confirm 🙂

Reply:
This would make sense. Let me know if I can assist 👍

Reply:
Hello, just implemented the changes! It should indeed be diff instead of list, sorry for not catching this earlier :)

Reply:
Thank you!